mantle: work around for ssh crypto policies change

dustymabe opened the pull request:
In 5c036d1 we swtched to ecdsa keys for the new crypto
policy change. In fdcfbbc we switched it back because
when we create new ssh keys on AWS we'd get an error because
that platform only accepts rsa keys:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#how-to-generate-your-own-key-and-import-it-to-aws

For now let's disable the `fcos.ignition.v3.noop` on AWS and
just use a fake key for that platform to satisfy the requirement.
Eventually we'll switch back to RSA keys once the golang library
has been updated.

See https://github.com/coreos/coreos-assembler/issues/1772

dustymabe commented:
this should obsolete #1797

dustymabe commented:
When we the golang change lands upstream and we update our golang library we can just wholesale revert the send commit in this PR and go back to the way things were before.

bgilbert reviewed (approved):
Content LGTM.  The second commit message should probably mention explicitly that we're switching back to ECDSA for now.

dustymabe commented:
> Content LGTM. The second commit message should probably mention explicitly that we're switching back to ECDSA for now.

update the commit message:

```
mantle: work around for ssh crypto policies change

In 5c036d1 we switched to ecdsa keys for the new crypto
policy change. In fdcfbbc we switched it back because
when we create new ssh keys on AWS we'd get an error because
that platform only accepts rsa keys:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#how-to-generate-your-own-key-and-import-it-to-aws

For now let's disable the `fcos.ignition.v3.noop` on AWS, use
a fake key for that platform to satisfy the requirement, and use
an ECDSA key everywhere else. Eventually we'll switch back to
RSA keys once the golang library has been updated.

See https://github.com/coreos/coreos-assembler/issues/1772

```

dustymabe commented:
update:

```
mantle: work around for ssh crypto policies change

In 5c036d1 we switched to ecdsa keys for the new crypto
policy change. In fdcfbbc we switched it back because
when we create new ssh keys on AWS we'd get an error because
that platform only accepts rsa keys:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#how-to-generate-your-own-key-and-import-it-to-aws

For now let's disable the `fcos.ignition.v3.noop` on AWS, use
a fake key for that platform to satisfy the requirement, and use
an ECDSA key on other clouds and in all Ignition configs. Eventually
we'll switch back to a single RSA key once the golang library
has been updated.

See https://github.com/coreos/coreos-assembler/issues/1772

```

bgilbert commented:
/lgtm
